packages/caliper-ethereum/test/connectorFactory.js

```diff
@@ -0,0 +1,51 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
```

Comment:
We need to do more in this unit test file, for example in the ethereum constructor it will do some basic validation of the network configuration. We should have tests there that test different network configurations.

Reply:
Added corresponding test suites in `packages/caliper-ethereum/test/ethereun-connector.js`